### PR TITLE
return of the code phrase 2: return of the return of the code phrase

### LINKED ISSUE #20763: syndicate code phrases are dead again
#### Description of issue

I rolled syndie twice, zero codewords. Don't think nukies get them either, though dunno if they ever did.
Maybe tie them to the syndicate faction?
Maybe give them to nukies while you're at it so they could (theoretically) interact with agents that are already on-station?
[roleissue] [bugfix]

#### Difference between expected and actual behavior
HELLO SYNDIE, HERE'S UR CODE, HERE ARE YOUR JECTIES, HERE ARE YOUR CODEWORDS
vs
HELLO SYNDIE, HERE'S UR CODE, HERE ARE YOUR JECTIES

#### Steps to reproduce
Spawn as syndie, make someone a syndie, etcetera. 
Observe greeting / notes

#### Specific information for fixing
`syndicate_code_phrase`
Easiest, gayest fix would be to just add the above var to the syndie greeting, like
```
The Syndicate provided you with the following information on how to identify their agents:
Code Phrase: [syndicate_code_phrase]
Code Response: [syndicate_code_response]
Use the code words in the order provided, during regular conversation, to identify other agents. Proceed with caution, however, as everyone is a potential foe.
```

with a 

```
mind.store_memory("<b>Code Phrase</b>: [syndicate_code_phrase]")
mind.store_memory("<b>Code Response</b>: [syndicate_code_response]")
``` 

in the setup.

#### Length of time in which bug has been known to occur
Can't specify, but probably since datums.

#### Server revision
46405f56b416a3da2232344c3a4fa4e0db625a79

### PR DESCRIPTION
**Pull request by SonixApache**
closes #20763

🆑 
 - rscadd: Syndicate agents are now once again provided with code phrases to help identify eachother.

### CODE REVIEW

**Comment by Exxion** [comment]
Best PR title of the year

**Comment by SonixApache** [comment]
it's a code phrase

File: code/datums/gamemode/misc_gamemode_procs.dm [hunk 1]
```diff
@@ -271,3 +271,28 @@ proc/name_wizard(mob/living/carbon/human/wizard_mob)
...
+			agent.mind.store_memory("<b>Code Response</b>: [syndicate_code_response]")
+	else
+		words += "Unfortunately, the Syndicate did not provide you with a code response.<br>"
+		
+	if(syndicate_code_phrase || syndicate_code_response)
+		words += "Use the code words in the order provided, during regular conversation, to identify other agents. Proceed with caution, however, as everyone is a potential foe.<br>"
+	else
+		words += "Trust nobody.<br>"
+		
+	if(agent)
```
Comment (MadmanMartian) [inline code/datums/gamemode/misc_gamemode_procs.dm]: surely it'd be better to check if there's an agent at the start, else what's the point of everything above?

Comment (SonixApache) [reply code/datums/gamemode/misc_gamemode_procs.dm]: So you can run the proc without a var?
Or maybe if the guy disappears mid-proc? I don't really know anymore.

**Pull request merged**